shared/constants/archive.tsx

```diff
@@ -84,6 +146,27 @@ export const _useState = Z.createZustand<State>((set, get) => {
         }
       })
     },
+    cancelOrDismissKBFS: async (jobID: string) => {
```

Comment:
i'd make this not async and make an async helper inside that you `ignorePromise` on, currently this return Promise<void> vs void due to this